ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceDownloadQueue.swift

```diff
@@ -414,8 +414,6 @@ class ResourceDownloadQueue: HTTPDownloadDelegate {
         // The request has succeeded.
         if downloader!.requestsCount == 0 { // Download queue finished.
           let keyboards = batch.resources as! [InstallableKeyboard]
-
-          FontManager.shared.registerCustomFonts()
           log.info("Downloaded keyboard: \(keyboards[0].id).")
```

Comment:
Is this for downloading cloud keyboards (.js) files and fonts from s.keyman.com?
Where did registerCustomFonts() get refactored to?

Reply:
It's always called on `Manager.shared.addKeyboard()` calls, among other places.  This single call occurred before any new fonts actually got installed and was out of place.  I didn't touch any of its other calls.